test_gpios.py

```diff
@@ -129,6 +129,21 @@ async def record_tick_data(tick_stream, data, should_stop):
             if should_stop.is_set():
                 return
 
+    async def test_monitor(self):
+        """
+        One board canary is designated the "canary monitor," and makes sure
+        that all other canaries are online, reachable, and have run their
+        analysis in the past day. There is a different canary that's in charge
+        of making sure the canary monitor is online, and that's what this test
+        is about.
+        """
+        if config.ssh_monitor is None:
+            return  # No monitor to ping
+
+        subprocess_result = subprocess.run(
```

Comment:
I feel like its a little confusing that the actual canary monitor is a cron job but this is done in the test file. Could we have the actual canary monitor ping the other boards in a function in this file so we dont need multiple cron jobs? (maybe this isn't possible just wondering)